src/interfaces/wallet.h

```diff
@@ -320,31 +320,31 @@ class WalletLoader : public ChainClient
 {
 public:
     //! Create new wallet.
-    virtual std::unique_ptr<Wallet> createWallet(const std::string& name, const SecureString& passphrase, uint64_t wallet_creation_flags, bilingual_str& error, std::vector<bilingual_str>& warnings) = 0;
+    virtual util::Result<std::unique_ptr<Wallet>> createWallet(const std::string& name, const SecureString& passphrase, uint64_t wallet_creation_flags, std::vector<bilingual_str>& warnings) = 0;
```

Comment:
Would probably be best to also wrap the warnings in the result? Are you working on this @ryanofsky ?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/25616#discussion_r942704207

> Would probably be best to also wrap the warnings in the result? Are you working on this @ryanofsky ?

It needs rebase, but yes I did this in #25722

Reply:
Yeah, I meant a minimal extract without the other C++ bloat (for example `Result<void>`) that isn't needed for simply passing warnings